Python/Product/Analysis/Parsing/Parser.cs

```diff
@@ -4197,8 +4211,25 @@ struct Name {
 
         private bool PeekTokenForOrAsyncFor {
```

Comment:
Alternative is change it to `PeekTokenForOrAsyncFor (bool inGeneratorExpression)`. Up to you.

Reply:
I started going that way and didn't like itp